cellom2tif/filetypes.py

```diff
@@ -1,3 +1,32 @@
+import os
+
+
+def fn_has_ext(fn, ext, case_sensitive=False):
```

Comment:
Yeah, sure, simplicity is good and it's an easy enough PR to fix if we ever need it. But do write it down clearly, ideally in several places (e.g. in the first line of the docstring and also in the return value description), so that users are not surprised when their .dib files match .DIB.